lib/ContainerManager.js

```diff
@@ -23,6 +25,17 @@ Promise.longStackTraces();
 
 const SETUP_CONTEXT = "env"
 
+var log_container_events = function (container){
+  var events = ["running tasks", "tasks complete", "creating", "stopping", "restarting",
```

Comment:
If we need to act on all events for a particular event emitter I wonder wether we should either A. make a more generic event that is emitted along with the other events or B. use [EventEmitter2](https://github.com/asyncly/EventEmitter2) which would allow us to do a wildcard bind.

This array violates the principle of least knowledge as, correct me if I'm wrong, we're binding on all events of the container and as such any time we add an event to the container it also needs to be added here. I wonder if we should have a more generic *stateChange* event (maybe in addition to the more specialized events) and then bind logging to that.